doc/api/deprecations.md

```diff
@@ -3,9 +3,11 @@
 <!--introduced_in=v7.7.0-->
 <!-- type=misc -->
 
-Node.js may deprecate APIs when either: (a) use of the API is considered to be
-unsafe, (b) an improved alternative API is available, or (c) breaking changes to
-the API are expected in a future major release.
+Node.js may deprecate APIs when:
```

Comment:
@trott How about the following suggestion to avoid confusion?
```suggestion
Node.js may deprecate APIs when one or more of the following:
```

Reply:
I see what you're going for. Maybe this?:

```suggestion
Node.js may deprecate APIs for any of the following reasons:
```

Reply:
Done!